lib/src/checks.dart

```diff
@@ -1,4 +1,48 @@
+import 'package:nyxx/nyxx.dart';
 import 'package:nyxx_commands/nyxx_commands.dart';
+import 'package:running_on_dart/src/services/music.dart';
 import 'package:running_on_dart/src/settings.dart';
 
 final administratorCheck = UserCheck.anyId(adminIds, 'Administrator check');
+
+final connectedToAVoiceChannelCheck = Check((IContext context) async {
+  final selfMember = await context.guild!.selfMember.getOrDownload();
+
+  if (selfMember.voiceState == null || selfMember.voiceState!.channel == null) {
+    throw MusicCheckException(context, 'I have to be in a voice channel to use this command');
```

Comment:
By convention, checks shouldn't throw an exception but return `false` to indicate failure. The check can then be identified in the error handler using the optional `name` parameter on the `AbstractCheck` class in order to send the correct response.

The reason for this is simply that this will throw an `UncaughtException` instead of a `CheckFailedException`.

Reply:
Well, i don't want the user to be sent the normal `an error occurred` embed because can be confusing, so that's why i'm throwing a custom exception there and catching it to send the user what happened

Reply:
Well, i've tried to do so, but the name on the `AbstractCheck` class just returns `Check`

![imagen](https://user-images.githubusercontent.com/62391364/173922204-fc1d9b98-b03c-45d6-841b-b28996ff32a6.png)

Reply:
To give a check a name, you must give it one when you create it as the second positional argument in the constructor. The default is just `"Check"`, which is what you saw.